app/src/androidTest/java/ch/epfl/sweng/swenggolf/CategoriesMenuTest.java

```diff
@@ -38,16 +38,16 @@
     @Rule
     public final IntentsTestRule<MainMenuActivity> activityRule =
             new IntentsTestRule<>(MainMenuActivity.class, false, false);
+    private static final Offer offer1 = (new Offer.Builder()).setUserId("user_id")
+            .setTitle("This is a title").setDescription(LOREM)
+            .setUuid("idoftheoffer1").build();
```

Comment:
That's just a general comment and it is not at all directed towards you; but I think it would be wise to harmonize all tests (e.g. for the next sprint) to make full use of the wonderful `FilledFakeDatabase` made by @udamiano. Do you agree with me?

Reply:
I think it's a good idea to use the <code>FilledFakeDatabase</code> for the new tests. That what I always do. Here I was just modifying an existing test, it was faster to reuse what was already here.
In the future, it is a good idea to use this database when we modify tests. But I don't think it's a good use of our time to change all the tests. It would be better spent on adding new features or enhancing existing ones.

Reply:
I feel the same way as @SFulpius it's pointless to go over all the tests and change them but we have to bear in mind that the FilledFakeDatabaseExists.